SU2_CFD/include/variable_structure.inl

```diff
@@ -858,6 +783,122 @@ inline void CAdjNSVariable::SetVelSolutionOldDVector(void) { for (unsigned short
+
+inline void CIncEulerVariable::SetLimiter_Primitive(unsigned short val_var, su2double val_value) { Limiter_Primitive[val_var] = val_value; }
+
+inline su2double **CIncEulerVariable::GetGradient_Primitive(void) { return Gradient_Primitive; }
+
+inline su2double *CIncEulerVariable::GetLimiter_Primitive(void) { return Limiter_Primitive; }
+
+inline void CIncEulerVariable::SetTimeSpectral_Source(unsigned short val_var, su2double val_source) { TS_Source[val_var] = val_source; }
+
+inline su2double CIncEulerVariable::GetTimeSpectral_Source(unsigned short val_var) { return TS_Source[val_var]; }
+
+inline void CIncEulerVariable::SetWindGust( su2double* val_WindGust) {
+  for (unsigned short iDim = 0; iDim < nDim; iDim++)
+    WindGust[iDim] = val_WindGust[iDim];}
+
+inline su2double* CIncEulerVariable::GetWindGust() { return WindGust;}
+
+inline void CIncEulerVariable::SetWindGustDer( su2double* val_WindGustDer) {
+  for (unsigned short iDim = 0; iDim < nDim+1; iDim++)
+    WindGustDer[iDim] = val_WindGustDer[iDim];}
+
+inline su2double* CIncEulerVariable::GetWindGustDer() { return WindGustDer;}
+
+inline su2double CIncNSVariable::GetEddyViscosity(void) { return Primitive[nDim+4]; }
+
+inline su2double CIncNSVariable::GetLaminarViscosity(void) { return Primitive[nDim+3]; }
+
+inline su2double* CIncNSVariable::GetVorticity(void) { return Vorticity; }
+
+inline su2double CIncNSVariable::GetStrainMag(void) { return StrainMag; }
+
+inline void CIncNSVariable::SetLaminarViscosity(su2double val_laminar_viscosity_inc) { Primitive[nDim+3] = val_laminar_viscosity_inc; }
+
+inline void CIncNSVariable::SetEddyViscosity(su2double eddy_visc) { Primitive[nDim+4] = eddy_visc; }
+
+inline su2double *CAdjIncEulerVariable::GetForceProj_Vector(void) { return ForceProj_Vector; }
+
+inline su2double *CAdjIncEulerVariable::GetObjFuncSource(void) { return ObjFuncSource; }
+
+inline su2double *CAdjIncEulerVariable::GetIntBoundary_Jump(void) { return IntBoundary_Jump; }
+
+inline void CAdjIncEulerVariable::SetForceProj_Vector(su2double *val_ForceProj_Vector) { for (unsigned short iDim = 0; iDim < nDim; iDim++) ForceProj_Vector[iDim] = val_ForceProj_Vector[iDim]; }
+
+inline void CAdjIncEulerVariable::SetObjFuncSource(su2double *val_ObjFuncSource) { for (unsigned short iVar = 0; iVar < nVar; iVar++) ObjFuncSource[iVar] = val_ObjFuncSource[iVar]; }
+
+inline void CAdjIncEulerVariable::SetIntBoundary_Jump(su2double *val_IntBoundary_Jump) { for (unsigned short iVar = 0; iVar < nVar; iVar++) IntBoundary_Jump[iVar] = val_IntBoundary_Jump[iVar]; }
+
+inline void CAdjIncEulerVariable::SetPhi_Old(su2double *val_phi) { for (unsigned short iDim = 0; iDim < nDim; iDim++) Solution_Old[iDim+1]=val_phi[iDim]; };
+
+inline void CAdjIncEulerVariable::SetTimeSpectral_Source(unsigned short val_var, su2double val_source) { TS_Source[val_var] = val_source; }
+
+inline su2double CAdjIncEulerVariable::GetTimeSpectral_Source(unsigned short val_var) { return TS_Source[val_var]; }
+
```

Comment:
One additional note is that I am not sure HB incompressible would work like it is now, we can however create a testcase and if needed do the proper modifications.